example/v1/types_stable.go

```diff
@@ -67,6 +67,14 @@ type StableConfigTypeSpec struct {
 	// celUnion demonstrates how to validate a discrminated union using CEL
 	// +optional
 	CELUnion CELUnion `json:"celUnion,omitempty"`
+
+	// EvolvingCollection demonstrates how to have a collection where the maximum number of items varies on cluster type.
```

Comment:
Nit, comments should start with json tag version
```suggestion
	// evolvingCollection demonstrates how to have a collection where the maximum number of items varies on cluster type.
```

Reply:
Will fix.  Thanks!